src/App.tsx

```diff
@@ -37,6 +38,7 @@ function App() {
                 <Route path="/referral/upload" element={<MenuReferralUpload/>}/>
                 <Route path="/register" element={<RegisterLegal/>}/>
                 <Route path="/register/family-members" element={<RegisterFamilyMembers/>}/>
+                <Route path="/register/family-members/add" element={<AddFamilyMember/>}/>
```

Comment:
In the future I would add a piece of refactoring to make paths constants. It would avoid typos etc.

and it let's you unit tests paths easily

the above path could be:

```
const BASE_PATH_REGISTER = "/register";
const PATH_FAMILY_MEMBERS = "/family-members";
const PATH_ADD = "/add";

<Route path= `${BASE_PATH_REGISTER}/${PATH_FAMILY_MEMBERS}/${PATH_ADD}` element={<AddFamilyMember/>}/>
```